operators/src/processing/reprojection.rs

```diff
@@ -219,11 +234,25 @@ impl RasterOperator for Reprojection {
         let in_desc: &RasterResultDescriptor = raster_operator.result_descriptor();
         let out_no_data_value = in_desc.no_data_value.unwrap_or(0.); // TODO: add option to force a no_data_value
 
+        let bbox = if let Some(bbox) = in_desc.bbox {
+            let in_srs: Option<SpatialReference> = in_desc.spatial_reference.into();
+            let projector = CoordinateProjector::from_known_srs(
+                in_srs.ok_or(Error::AllSourcesMustHaveSameSpatialReference)?,
+                self.params.target_spatial_reference,
+            )?;
+
+            Some(bbox.reproject(&projector)?)
+        } else {
+            None
+        };
```

Comment:
it's not the same code because one time it's a BoundingBox and the other time a SpatialPartition. I doubt it will save much code to introduce a generic method.